vars/folioHelm.groovy

```diff
@@ -222,6 +222,8 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
   if (enableIngress) {
     moduleConfig['ingress']['hosts'][0] += [host: domain]
     moduleConfig['ingress']['annotations'] += ['alb.ingress.kubernetes.io/group.name': "${ns.clusterName}.${ns.namespaceName}"]
+    moduleConfig['ingress']['annotations'] += ['alb.ingress.kubernetes.io/target-type': 'ip']
```

Comment:
Will it work on ip type of routing?

Reply:
Yep, validated https://jenkins-aws.indexdata.com/job/folioRancher/job/tmpFolderForDraftPipelines/job/OleksandrH/job/createNamespaceFromBranch-RANCHER-1764/32/